data/wanderers start.txt

```diff
@@ -376,16 +376,59 @@ mission "Tell Wanderers about the Pug"
 				decline
 
 
+mission "Wanderers: Defend Vara Ke'sok Hint"
+	name "Wanderer supplies to <planet>"
+	description "Transport <cargo> to <system>, where repairs are needed from Unfettered attacks."
+	passengers 1
+	cargo "building supplies" 20
```

Comment:
I'd say 10 tons just so it's more likely that the player has space for this.

Reply:
Changed to 10 tons